client/src/components/BottomNav.css

```diff
@@ -0,0 +1,31 @@
+footer {
+  position: absolute;
+  inset: auto 0 0 0;

```

Comment:
What does this do?

Reply:
It's a shorthand, one-line-solution for: 

top: auto; 
right: 0; 
left: 0; 
bottom: 0;